plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core_aws.py

```diff
@@ -115,19 +115,20 @@ def launch(
 def launch_jobs(
     launcher: RayAWSLauncher, local_tmp_dir: str, sweep_dir: Path
 ) -> Sequence[JobReturn]:
-    ray_up(launcher.ray_yaml_path)
+    ray_up(launcher.ray_yaml_path, launcher.ray_cfg.no_config_cache)
     with tempfile.TemporaryDirectory() as local_tmp_download_dir:
 
         with ray_tmp_dir(
-            launcher.ray_yaml_path, launcher.docker_enabled
+            launcher.ray_yaml_path, launcher.ray_cfg.run_env.name
         ) as remote_tmp_dir:
 
             ray_rsync_up(
                 launcher.ray_yaml_path, os.path.join(local_tmp_dir, ""), remote_tmp_dir
             )
 
             script_path = os.path.join(os.path.dirname(__file__), "_remote_invoke.py")
-            ray_rsync_up(launcher.ray_yaml_path, script_path, remote_tmp_dir)
+            remote_script_path = os.path.join(remote_tmp_dir, "_remote_invoke.py")
+            ray_rsync_up(launcher.ray_yaml_path, script_path, remote_script_path)
```

Comment:
why this change? this only rsync up one certain file, there's other stuff in the dir that needs to be rsyned if I remember it correctly..

Reply:
that change was made to fix a FileNotFoundError I was getting in my dockerized app.

Reply:
I'm not sure this is a proper fix. What was the exact exception in the docker app?

Reply:
It happened during the `ray exec` call. `<remote_tmp_dir/_remote_invoke.py` no such file or directory

Reply:
I don't think this line is the problem. If you look at the circleCI output, the node can find `_remote_invoke.py` just fine and all this line did before was copy that specific script

```
[2021-01-07 00:55:26,487][HYDRA] Running command: ['ray', 'exec', '--run-env=auto', '/tmp/tmpb7uftols.yaml', 'python /tmp/tmp.AjwSf1fZyK/_remote_invoke.py /tmp/tmp.AjwSf1fZyK']
[2021-01-07 00:55:34,167][HYDRA] Output: 2021-01-07 00:55:28,634	INFO services.py:1090 -- View the Ray dashboard at http://127.0.0.1:8266
Traceback (most recent call last):
  File "/tmp/tmp.AjwSf1fZyK/_remote_invoke.py", line 87, in <module>
    launch_jobs(sys.argv[1])
  File "/tmp/tmp.AjwSf1fZyK/_remote_invoke.py", line 62, in launch_jobs
    result = [ray.get(run) for run in runs]
  File "/tmp/tmp.AjwSf1fZyK/_remote_invoke.py", line 62, in <listcomp>
    result = [ray.get(run) for run in runs]
  File "/home/ubuntu/anaconda3/envs/hydra_3.8.5/lib/python3.8/site-packages/ray/worker.py", line 1452, in get
    raise value.as_instanceof_cause()
ray.exceptions.RayTaskError(FileNotFoundError): ray::_run_job() (pid=6899, ip=172.31.35.143)
  File "python/ray/_raylet.pyx", line 482, in ray._raylet.execute_task
  File "/home/ubuntu/anaconda3/envs/hydra_3.8.5/lib/python3.8/site-packages/hydra_plugins/hydra_ray_launcher/_launcher_util.py", line 40, in _run_job
    return run_job(
  File "/home/ubuntu/anaconda3/envs/hydra_3.8.5/lib/python3.8/site-packages/hydra/core/utils.py", line 126, in run_job
    ret.return_value = task_function(task_cfg)
  File "/tmp/hydra_test/no_config.py", line 9, in experiment
  ```

Reply:
this error is being through _inside_ the `_remote_invoke.py` script, looking for something in an entirely different directory.